src/NodaTime.Web/wwwroot/lib/bootstrap/.bower.json

```diff
@@ -1,44 +0,0 @@
-{
```

Comment:
We aren't actually using any of the Bower-sourced files at all, so we can actually remove all of `wwwroot/lib`, `.bowerrc`, and `bower.json` (the latter still refers to bootstrap).